data/json/monsters.json

```diff
@@ -210,7 +210,7 @@
     "vision_day":40,
     "hp":16,
     "death_function":"NORMAL",
-    "description":"A spotted wild cat living across much of North America.  It is not a serious threat to humans, but it can be aggressive if not left alone.",
+    "description":"A spotted wild cat, these live across much of North America.  It is not a serious threat to humans, but it can be aggressive if not left alone.",
```

Comment:
Alternatively you could replace the comma with a semi-colon which might make it a bit more clear about the division in the phrasing, i.e.: "A spotted wild cat; these live across much of North America".